adobe_vipm/flows/fulfillment/shared.py

```diff
@@ -710,6 +711,29 @@ def __call__(self, client, context, next_step):
         next_step(client, context)
 
 
+class GetPreviewOrder(Step):
+    """
+    Retrieve a preview order for the upsize lines. Switch the order to failed
+    in case of exceptions.
+    """
+
+    def __call__(self, client, context, next_step):
+        adobe_client = get_adobe_client()
+        if context.upsize_lines and not context.adobe_new_order_id:
```

Comment:
Can you please put more comments on why we have this here? It is not clear, especially because your are using this  step only for purchase pipe